src/App.css

```diff
@@ -152,9 +152,8 @@ figcaption {
 
 .component-blog {
```

Comment:
We need to add new class for tutorials and leave this one with the previous version of it.